### ISSUE TITLE
Unformatted absolute deltas

### ISSUE THREAD

**Issue by perezju** [catapult-project/catapult#3105]
Some numbers in the new "abs delta" column in chrome perf show up as "unformatted"

![ndyuvhtv9cu](https://cloud.githubusercontent.com/assets/16922344/21424620/ea18dc7a-c83b-11e6-8106-a07ad249f575.png)

Why is that? Maybe a bug in our metrics not providing the correct unit?

@jessimb

**Comment by perezju** [comment]
Those show up in the second group of [alerts for this bug](https://chromeperf.appspot.com/group_report?bug_id=674924).

**Comment by jessimb** [comment]
This is WAI.

Unit was not information previously stored in the Anomaly entity, but it was stored in the Test entity that the Anomaly keeps a reference to. Rather than fetching from Anomaly and Test each time, we now store units in the Anomaly directly. I wrote a MapReduce job to add units to existing anomalies, but it didn't update all of them for some unknown reason. We determined that it wasn't worth our time to continue digging into the MapReduce problem when all new Anomalies created since Dec 13 should contain unit information.  

In the meantime, some older Anomalies may not have units and will appear (unformatted).
